cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/GetController.java

```diff
@@ -96,21 +93,4 @@ public Integer call() throws Exception {
             return 0;
         }
     }
-
-    @Command(name = "all", description = "show all information in the target space")
-    static class GetAllInformation implements Callable<Integer> {
-        @Mixin
-        LoginCommandOptions loginOptions;
-
-        @Override
-        public Integer call() throws Exception {
-            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
-            GetLogic getLogic = new GetLogic(cfOperations);
-            Log.info("Fetching all information for target space...");
-            ConfigBean allInformation = getLogic.getAll();
-
-            System.out.println(YamlMapper.dump(allInformation));
-            return 0;
-        }
-    }
```

Comment:
We should not remove the GetAllInformation class because this is not desired in the user story.